os/binfmt/libelf/Kconfig

```diff
@@ -55,3 +55,37 @@ config ELF_EXCLUDE_SYMBOLS
 	---help---
 		If this option is enabled, then it excludes symbol information from the ELF
 		and results in a ELF of much smaller size.
+
+config ELF_CACHE_READ
+        bool "ELF cache read support"
+        default n
+        depends on BINFMT_ENABLE
+        depends on !COMPRESSED_BINARY
+        ---help---
+		Enabling this config would increase the elf read performance by
+		manyfolds with its caching mechanism.
+
+		If an elf_read taking more time and consistently trying to read the
+		same position multiple times, then there would be a considerable delay.
+		Enabling this config will cache/buffer the previously accessed data.
+
+		Note: If COMPRESSED_BINARY=y, then this config should be disabled, as
+		by default COMPRESSED_BINARY config would support caching inbuilt.
```

Comment:
do we need to mention the Note? as by default it will be disabled if COMPRESSED_BINARY is enabled